### PR TITLE
Readthedocs.io

### PR DESCRIPTION
**Pull request by DanHickstein**
Following the discussion in #174, I changed all mentions of "readthedocs.org" to "readthedocs.io". The new URL is live, so I think that this should not cause any problems.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -107,20 +107,20 @@ The implementation named `<implementation>`, located under `abel/<implementation
  -  ``<implemenation>_transform`` :  core transform (when defined)
  -  ``_bs_<implementation>`` :  function that generates  the basis sets (if necessary)
  -  ``bs_<implementation>_cached`` : function that loads the basis sets from disk, and generates them if they are not found (if necessary).
```
Comment (stggh) [inline CONTRIBUTING.rst]: `bs_<implementation>_cached` is actually a common function `get_bs_cached(method, ...)` located in `abel.tools.basis.py`, for Dasch and Gerber methods, and for `basex` the function `abel.basex.get_bs_basex_cached()`. The `basex` version may easily be incorporated into `basis.py` (once I, or someone else has a little spare time).  The reason for `get_bs_cached` is just the way it was done in `basex`.

So, perhaps line 109 may read:
```
``abel.tools.basis.py`` (if necessary) modify this file to provide for loading the basis sets from disk, or calling the generator function
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -56,15 +56,15 @@ Now you should be able to ::
...
 Alternatively, `restview <https://pypi.python.org/pypi/restview>`_ is a nice way to preview the ``.rst`` files.
 
 Before merging
 --------------
 
 If possible, before merging your pull request please rebase your fork on the last master on PyAbel. This could be done `as explained in this post <https://stackoverflow.com/questions/7244321/how-to-update-a-github-forked-repository>`_ ::
-   
+
     # Add the remote, call it "upstream" (only the fist time)
     git remote add upstream git@github.com:PyAbel/PyAbel.git
```
Comment (stggh) [inline CONTRIBUTING.rst]: I may be alone with this, but I have found that only the address given on the home page works:
`https://github.com/PyAbel/PyAbel.git`

**Comment by stggh** [comment]
@DanHickstein good work finding more than the single change originally identified.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (DanHickstein) [inline CONTRIBUTING.rst]: This makes sense. I can make this change.

**Comment by DanHickstein** [comment]
@stggh: I made the changes that you suggested to CONTRIBUTING.rst. If these seem okay, you can press the green button.

For the record, I am still a little bit confused about the whole "rebasing" thing. I have always just hit the "update from PyAbel/master" button in the GitHub Desktop GUI application. I think that this basically commits the recent changes from the repo into my fork/branch. Is there a reason to suggest this rebasing method rather than also suggesting that people use the GitHub desktop application?

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -66,7 +66,7 @@ Before merging
 If possible, before merging your pull request please rebase your fork on the last master on PyAbel. This could be done `as explained in this post <https://stackoverflow.com/questions/7244321/how-to-update-a-github-forked-repository>`_ ::
 
     # Add the remote, call it "upstream" (only the fist time)
-    git remote add upstream git@github.com:PyAbel/PyAbel.git
+    git remote add upstream git@https://github.com/PyAbel/PyAbel.git
```
Comment (stggh) [inline CONTRIBUTING.rst]: This should be just:   https://github.com/PyAbel/PyAbel.git

**Comment by stggh** [comment]
@DanHickstein thanks. Note the address should be just:   https://github.com/PyAbel/PyAbel.git

As a command-line jockey I find the rebase instructions in `CONTRIBUTING.rst` to be essential. However, there is nothing wrong with choice, so if you like add the "update from PyAbel/master" button. I cannot say that I have noticed it.  

Also, I appear to be handing you trivial changes that I could make myself in the next `linbasex` PR. This is not intended, just the `readthedocs.io` update reminded me about what could be changed.

**Comment by DanHickstein** [comment]
@stggh, I made the tweak. I think that I recall that there is some reason why rebasing as currently described in the CONTRIBUTING.rst is better than what I'm doing (I think that it might result in a cleaner git history), so I left the guidance the way that it was. 

Yes, I'd say merge this PR and you can make additional changed in your other mega-PR.

**Pull request merged**